.versionrc

```diff
@@ -0,0 +1,7 @@
+{
+  "types": [
+    {"type":"feat","section":"Features"},
+    {"type":"fix","section":"Bug Fixes"},
+    {"type":"test", "hidden": true}
```

Comment:
I think we can have extra settings for `Features` and `Bug Fixes` and keep only one line for tests

Reply:
No, they are required.

Reply:
hm, okay, i think we need use this:
```
{
  "types": [
    {"type":"feat","section":"Features"},
    {"type":"fix","section":"Bug Fixes"},
    {"type":"test","section":"Tests", "hidden": true},
    {"type":"build","section":"Build System", "hidden": true},
    {"type":"ci","hidden":true}
  ]
}
```

Reply:
Why? We don't use build and CI. They are unnecessary.

Reply:
i am use `ci` and `build`, but very rare, it is interesting, why standard-release doesn't generate `chore`, but generate `test:`

Reply:
Anyway thanks for investigate, it is really very strange

Reply:
np, I made a mistake to forget to retag when I deleted test's section from CHANGELOG.md and rebased.(v3.6.0)

Reply:
Anyway, we use `ci` and `build` in our repo, please add these. I'll merge this pr.